components/service/glean/build.gradle

```diff
@@ -14,7 +14,7 @@ apply plugin: 'kotlin-android'
  * created during unit testing.
  * This uses a specific version of the schema identified by a git commit hash.
  */
-String GLEAN_PING_SCHEMA_GIT_HASH = "3bf541d"
+String GLEAN_PING_SCHEMA_GIT_HASH = "2d5c56f3d95f34c2681babb38e6b06d0bdfed087" // TODO update this back to master git hash when https://github.com/mozilla-services/mozilla-pipeline-schemas/pull/305 lands
```

Comment:
Let's make sure to NOT merge this until schemas are updated and this gets changed.

Reply:
Of course.  I did update it just now to point to an update in the schema PR that is still open just to verify it still passed all tests with the addition to that repo.  

Once the schema lands I'll get the correct git hash in here and land this one.